Formula/z/zig.rb

```diff
@@ -42,6 +43,19 @@ def install
     system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
     system "cmake", "--build", "build"
     system "cmake", "--install", "build"
+
+    return unless OS.linux?
+
+    bin_to_patch = bin/"zig"
+    extra_rpath = [
+      rpath(target: Formula["z3"].opt_lib),
+      rpath(target: Formula["zstd"].opt_lib),
+      rpath(target: Formula["ncurses"].opt_lib),
+      rpath(target: Formula["zlib"].opt_lib),
+    ]
+    patcher = bin_to_patch.patchelf_patcher
+    patcher.rpath = [*bin_to_patch.rpaths, extra_rpath].join(":")
+    patcher.save(patchelf_compatible: true)
```

Comment:
This is quite unusual. The patching here shouldn't be needed because the `rpath`s should have already been set during link time. But without this, the `zig` executable has no `rpath`s at all.

Reply:
This is likely due to ziglang/zig#13466.